migrations/V155__Update_NDAAgreement_AddVersions.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE nda_agreement
+  RENAME COLUMN agreed TO v1_agreed;
+
+ALTER TABLE nda_agreement
+  ADD COLUMN v2_agreed BOOLEAN NOT NULL default false;
```

Comment:
I think we should probably have a separate column to capture the time the v2 was agreed to.